.taskcluster.yml

```diff
@@ -1,6 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 version: 0
 allowPullRequests: collaborators
 tasks:
+####################################################################################################
+# Task: Pull requests
+####################################################################################################
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        env: true
+        events:
+          - pull_request.opened
+          - pull_request.edited
+          - pull_request.synchronize
+          - pull_request.reopened
+          - push
+    scopes:
+      - "queue:create-task:aws-provisioner-v1/github-worker"
```

Comment:
I assume this is needed

Reply:
Yes -- without this, the decision task doesn't have access to the TC queue :)